src/bin/nydus-image/main.rs

```diff
@@ -698,21 +697,33 @@ fn prepare_cmd_args(bti_string: &'static str) -> App {
                         .required(false),
                 )
                 .arg(
-                    Arg::new("blob")
-                        .long("blob")
-                        .short('b')
-                        .help("Path to RAFS data blob file")
-                        .required(false)
-                        .group("backend"),
+                    Arg::new("min-used-ratio")
+                        .long("min-used-ratio")
+                        .help("Lower bound of used ratio for blobs to be kept")
```

Comment:
May be add some help hint: `Lower bound of used ratio for blobs to be kept, possible values: 0-99, 0 means disable`.